chord_metadata_service/mohpackets/permissible_values.py

```diff
@@ -3,11 +3,16 @@
 MoH model fields and are used for validation during serialization.
 """
 
+BOOLEAN = [
+    "Yes",
+    "No",
+    "Unknown"
+]
+
 CAUSE_OF_DEATH = [
```

Comment:
Please change the name to something else, status_indicator or something, boolean means binary.